scoresheets/Restaurant.tex

```diff
@@ -1,24 +1,17 @@
-The maximum time for this test is 15 minutes.
+The maximum time for this test is 10 minutes.
 
-\small\begin{scorelist}
+\small\begin{scorelist}[startbutton=false]
 	\scoreheading{Main Goal}
-	\scoreitem{500}{Take and serve an order}
-	\penaltyitem[2]{200}{Being guided to the \emph{Kitchen-bar} or to a customer's table}
-	\penaltyitem{100}{Not making eye-contact when taking an order}
-	\penaltyitem[4]{50}{Bypassing object manipulation (handover)}
-	\penaltyitem[5]{100}{Bypassing object manipulation}
+	\scoreitem[2]{500}{Take and serve an order}
+	\penaltyitem{25}{Request help by receiving directions}
+	\penaltyitem{200}{Request guidance to a table}
+	\penaltyitem[2]{350}{Deliver a partly wrong order}
+	\penaltyitem[2]{500}{Deliver a fully wrong order}
+	\penaltyitem[2]{100}{Handing over an order instead of placing}
 
-	\scoreheading{Second order (bonus reward)}
-	\scoreitem{500}{Take and serve an additional order}
-	\penaltyitem[2]{200}{Being guided to the \emph{Kitchen-bar} or to a customer's table}
-	\penaltyitem{100}{Not making eye-contact when taking an order}
```

Comment:
Just making sure that this was an intentional removal? I think it was a nice add-on. I we think that the test is too hard, this could be turned into a bonus when done instead than a malus when not done?

Reply:
Yes intentional. Eye contact is somewhat difficult to determine for some platforms.